test/client_add_user_properties_test.rb

```diff
@@ -92,6 +113,24 @@ def test_add_user_properties
         'foo' => 'bar', :heap => :hurray)
   end
 
+  def test_add_user_properties_integer_identity
+    identity = 123456789
+    @stubs.post '/api/identify' do |env|
+      golden_body = {
+        'app_id' => 'test-app-id',
+        'identity' => identity.to_s,
```

Comment:
I think I'd prefer hard-coding the identity as a string here and as a number below, to keep the test super-simple.